examples/miniapps/decoupled-packages/example/abstraction/photo/repositories.py

```diff
@@ -0,0 +1,11 @@
+"""Photo repositories Meta module."""
+
+import abc
+
+
+class PhotoRepositoryMeta(metaclass=abc.ABCMeta):
+
+    @abc.abstractmethod
+    def get_photos(self, user_id):
+        """Must be implemented in order to instantiate."""
+        pass
```

Comment:
```suggestion
```
Coverage report treats `pass` as a code. Which never executes for `abstractmethod`s.